astropy/cosmology/io/tests/test_mapping.py

```diff
@@ -190,7 +190,7 @@ def test_is_equivalent_to_mapping(self, cosmo, to_format, format):
         assert not isinstance(obj, Cosmology)
 
         is_equiv = cosmo.is_equivalent(obj, format=format)
-        assert is_equiv is (True if format is not False else False)
+        assert is_equiv is (bool(format is not False))
```

Comment:
This is bad, IMO. Doesn't `(format is not False)` already evaluate to a bool type?
```suggestion
        assert is_equiv is (format is not False)
```

Reply:
I believe it should. I can make this change manually. The code linter has a specific pattern: ``True if X else False`` that it knows to transform to ``bool(X)``, but is not combined with a static type checker to make further refinements.

Reply:
✅ Made the change